Source/Core/Editor/UI/Drawers/LockableObjectsDrawer.cs

```diff
@@ -188,5 +191,23 @@ private Rect DrawProperty(Rect currentPosition, LockableProperty property)
             EditorGUI.LabelField(objectPosition, property.GetType().Name);
             return currentPosition;
         }
+
+        private void DrawSearchableTagListPopup(Rect rect, Action<SceneObjectTags.Tag> onItemSelected, IEnumerable<Guid> tagsToExclude)
+        {
+            string searchableListPath = "Assets/MindPort/VR Builder/Core/Source/Core/Editor/UI/Views/SearchableList.uxml";
+            string tagListItemPath = "Assets/MindPort/VR Builder/Core/Source/Core/Editor/UI/Views/SearchableListItem.uxml";
```

Comment:
@VaLiuM09  Should we create a global file with references to to the .uxml files.
I guess we will need the searchable drop down in other places was well.

Reply:
Thanks!
Just to have no misunderstanding IMO the preferable way would be to reference them in the Editor. Like theses
![Screenshot 2024-02-23 082142](https://github.com/MindPort-GmbH/VR-Builder/assets/247111/3734986b-501d-40bd-b691-d9b34c53c8ca)
![Screenshot 2024-02-23 082059](https://github.com/MindPort-GmbH/VR-Builder/assets/247111/1003a27a-713f-410d-9f2e-94a870f09aa2)
But that is not possible for windows of type PopupWindowContent or for our Drawers. So when using them we have to resort back to the hardcoded path.

Also added the info the the ticket.